[WIP] Updating Keepers Documentation (#478)
smartcontractkit/documentation: docs/chainlink-keepers/introduction.md

To resolve this issue, blockchain projects can:

1. Create highly reliable infrastructure trigger smart contract functions. This is centralized and often expensive to build and maintain.
2. Outsource it to a third party. This option is also centralized and creates a single point of failure.
3. Use an open market solution. This option is decentralized, but comes with complex incentive alignment, the potential for competitive bots that increase the cost of execution, and difficulties in ensuring reliability.
4. Use the **Chainlink Keeper** Network

Chainlink Keepers provide users with a decentralized network of nodes that are incentivized to perform all registered jobs (or *Upkeeps*) without competing with each other. The network has many benefits, including the ability to:

- provide developers with hyper-reliable, decentralized smart contract automation
- offer expandable computation allowing developers to build more advanced dApps at lower costs
- be flexible and highly programmable

> We highly recommend reading the [Network Overview](../overview) page to learn how these networks work.

Reviewer: ```suggestion
Read the [Network Overview](../overview) page to learn how these networks work.
```

# Getting Started

> Our goal is to ensure flawless execution of Upkeeps when expected. To get the most out of Chainlink Keepers, we recommend you review these docs in full: [how to create a Keepers-compatible contract](../compatible-contracts), usage patterns, best practices, and how to [maintain the health of your upkeep](../register-upkeep).

Reviewer: ```suggestion
The goal of Keepers is to ensure flawless execution of Upkeeps when expected. To get the most out of Chainlink Keepers, review these docs in full: [how to create a Keepers-compatible contract](../compatible-contracts), usage patterns, best practices, and how to [maintain the health of your upkeep](../register-upkeep).
```

## Supported Blockchain Networks

Chainlink Keepers are currently available on Kovan and ETH testnet and mainnet. If you are interested in using Keepers but need it on a different blockchain or network, reach out to us and [let us know](https://forms.gle/WadxnzzjHPtta5Zd9).

## Onboarding Steps

> Below are the steps needed to create a Keeper compatible contract and register Upkeep for the contract. You should test Keeper-compatible contracts on testnet before moving to mainnet. 

Reviewer: ```suggestion
Below are the steps needed to create a Keeper compatible contract and register Upkeep for the contract. Test Keeper-compatible contracts on testnet before moving to mainnet. 
```

1. Create a [Keeper compatible contract](../compatible-contracts/)

> 2. Deploy your contract onto a supported testnet
> 3. Get [LINK](../../link-token-contracts/#kovan) on the testnet
> 4. Many token bridges give you ERC-20 LINK tokens. Use PegSwap to [convert Chainlink tokens (LINK) to be ERC-677 compatible](https://pegswap.chain.link/).
> 5. [Register your Upkeep](../register-upkeep/) on testnet
> 6. Test, iterate, and finalize your Keeper compatible contract
> 7. Deploy your fully tested contract onto mainnet
> 8. [Register your Upkeep](../register-upkeep/) on mainnet
> 9. Fund your upkeep on mainnet
> 10. Confirm the first performance of your Upkeep

Reviewer: ```suggestion
1. Deploy your contract onto a supported testnet
1. Get [LINK](../../link-token-contracts/#kovan) on the testnet
1. Many token bridges give you ERC-20 LINK tokens. Use PegSwap to [convert Chainlink tokens (LINK) to be ERC-677 compatible](https://pegswap.chain.link/).
1. [Register your Upkeep](../register-upkeep/) on testnet
1. Test, iterate, and finalize your Keeper compatible contract
1. Deploy your fully tested contract onto mainnet
1. [Register your Upkeep](../register-upkeep/) on mainnet
1. Fund your upkeep on mainnet
1. Confirm the first performance of your Upkeep
```

These should auto-order at build time.

> ⚠️ Registration Onboarding Note
> Your registrations on testnet will be auto approved immediately. To help us streamline the process of onboarding your use case on mainnet, please fill out the optional details (and form) during testnet registration so our onboarding team can expect your use case on mainnet. Mainnet registrations will be reviewed by our onboarding team before being approved. We are working towards a fully self-serve model.

> If you have questions, take a look at our [Keepers Frequently Asked Questions](../faqs/) page or [reach out to us](https://forms.gle/WadxnzzjHPtta5Zd9).

Reviewer: ```suggestion
If you have questions, read the [Keepers Frequently Asked Questions](../faqs/) page or [reach out to us](https://forms.gle/WadxnzzjHPtta5Zd9).
```
